american-journal-of-archaeology.csl

```diff
@@ -300,7 +317,7 @@
       <if variable="title" match="none"/>
       <else-if type="thesis"/>
```

Comment:
wouldn't you get duplicate genre here for presentations and other items with variable `event` if you don't exclude it?